src/manual_folds_with_custom_loss.py

```diff
@@ -273,15 +283,18 @@ def custom_basal_loss_with_inf(
     # %% this is where we can add in the 19 checks
     # this will look something like y_temp = equation(add in constants from our table (look at google doc)
     # y_temp needs to between min and max basal
-    for check_dict in basal_check_dicts:
+
+    for check_dict in bounding_check_dictionary:
         min_val = check_dict["MIN_OUTPUT"]
         max_val = check_dict["MAX_OUTPUT"]
 
         X_val = [check_dict[param] for param in X_col_names]
         y_pred = equation(fixed_parameters, X_val)
 
         if "log" in y_col_name:
+            warnings.filterwarnings('ignore')
```

Comment:
What is the effect of adding this call? Does it ignore a certain level of warnings?

Reply:
yes, it ignores warnings about really large numbers (if I remember correctly), which can happen with the use of the exponential function. I immediately turn the warnings back on two lines later, so I think this is OK.